wikipedia-dark.user.css

```diff
@@ -2998,11 +3011,25 @@ regexp("https?:\/\/wiki\.(archlinux|mozilla)\.(org|jp)\/.*$") {
   .oo-ui-textInputWidget.oo-ui-widget-enabled .oo-ui-inputWidget-input::placeholder {
     color: var(--gray-7) !important;
   }
-  .oo-ui-dropdownWidget.oo-ui-widget-enabled .oo-ui-dropdownWidget-handle,
-  .mw-mmv-label {
+  .mw-mmv-title-para .mw-mmv-ttf-ellipsis {
+    background-color: var(--gray-2);
+  }
+  .oo-ui-dropdownInputWidget.oo-ui-widget-enabled, .oo-ui-dropdownInputWidget.oo-ui-widget-enabled:hover {
+    background-color: transparent !important;
```

Comment:
Avoid `!important` where not absolutely needed, preferably increase specificity/weight of affected selectors

Reply:
I was unable to find a better solution.

Reply:
I asked for links to pages where the issue manifested so I could see about better solutions, =(